### PR TITLE
Added api_base, api_version params in openai llm

### PR DESCRIPTION
**Pull request by mohamed-siddhiq**
Added api_base, api_version params to the openai llm adapter

### CODE REVIEW

File: src/unstract/adapters/llm/open_ai/src/static/json_schema.json [hunk 1]
```diff
@@ -24,11 +24,24 @@
       "default": 5,
       "description": "The maximum number of times to retry a request if it fails."
     },
+    "api_base": {
```
Comment (chandrasekharan-zipstack) [inline src/unstract/adapters/llm/open_ai/src/static/json_schema.json:28]: NIT: @mohamed-siddhiq can you re-arrange the fields as below to have most relevant info on the top
1. adapter_name
1. api_key
1. model
1. api_base
1. api_version
1. max_retries
1. timeout

**Review by chandrasekharan-zipstack — CHANGES_REQUESTED** [review]
Can you bump the adapters version in the [pyproject.toml](https://github.com/Zipstack/unstract-adapters/blob/main/pyproject.toml#L8) - we could make it `0.1.2` I think since this isn't a breaking change. Do review and update the version as well

File: src/unstract/adapters/llm/open_ai/src/static/json_schema.json (hunk 1 above)
Comment (mohamed-siddhiq) [reply src/unstract/adapters/llm/open_ai/src/static/json_schema.json:28]: @chandrasekharan-zipstack  The fields are re-arranged.

**Review by johnyrahul — APPROVED** [review]
LGTM

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Pull request merged**